manifests/params.pp

```diff
@@ -93,6 +93,13 @@
 
   $messaging_client_cert = "${pki_dir}/qpid_client_striped.crt"
 
+  $qpid_dispatch_server_cert = "/etc/qpid-dispatch/server.crt"
+  $qpid_dispatch_client_cert = "/etc/qpid-dispatch/client.crt"
+  $qpid_dispatch_server_key  = "/etc/qpid-dispatch/server.key"
+  $qpid_dispatch_client_key  = "/etc/qpid-dispatch/client.key"
```

Comment:
Are these the dispatch router defaults or expected locations? i.e. are we able to configure the dispatch router where to look for certs ?